src/main/java/sirius/biz/tycho/academy/OnboardingController.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.tycho.academy;
+
+import sirius.biz.web.BizController;
+import sirius.kernel.commons.Tuple;
+import sirius.kernel.di.std.Part;
+import sirius.kernel.di.std.Register;
+import sirius.kernel.settings.Extension;
+import sirius.web.controller.Routed;
+import sirius.web.http.WebContext;
+import sirius.web.services.InternalService;
+import sirius.web.services.JSONStructuredOutput;
+
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Provides the UI for the onboarding / video academy framework.
+ */
+@Register
```

Comment:
Should this be bound to the framework?

Reply:
Nope, the routes have super low priorities and can easily overwritten if needed. If the OnboardingEngine is missing (no framework present), this will fail gracefully - as it shouldn't be called anyway).